test/external/imports/default-import/js-cjs/package.json

```diff
@@ -1,41 +1,21 @@
 {
-  "name": "express-rate-limit-example-js-cjs",
-  "version": "1.0.0",
-  "description": "A minimal example (JS-CJS) of a project using the express-rate-limit package.",
-  "scripts": {
-    "start": "node source/index.js",
-    "lint": "eslint source/**/*.js",
-    "test": "jest"
-  },
-  "dependencies": {
-    "express": "4.18.2",
-    "express-rate-limit": "file:../../../../.."
-  },
-  "devDependencies": {
-    "eslint": "8.51.0",
-    "jest": "29.7.0",
-    "supertest": "6.3.3"
-  },
-  "eslintConfig": {
-    "extends": "eslint:recommended",
-    "parserOptions": {
-      "ecmaVersion": 2022
-    },
-    "env": {
-      "es6": true,
-      "node": true
-    }
-  },
-  "jest": {
-    "verbose": true,
-    "collectCoverage": true,
-    "testTimeout": 30000,
-    "testMatch": [
-      "**/test/**/*-test.js"
-    ],
-    "moduleFileExtensions": [
-      "js",
-      "json"
-    ]
-  }
+	"name": "express-rate-limit-example-default-import-js-cjs",
```

Comment:
don't we have indent set to spaces?

Reply:
The `.editorconfig` file has only `*.yaml` files set to space so I went according to that. I could revert it to spaces if needed.